Packs/PaloAltoNetworks_SecurityAdvisories/ReleaseNotes/1_0_10.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks Security Advisories (Beta)
+
+- Updated the Palo Alto Networks Security Advisories integration to solve an issue where CVE data was not properly extracted.
```

Comment:
please change docs to be in the form of
fixed an issue where <command> did not extract CVE properly
or something of this sort.

Reply:
Fixed. Just worth noting that the SDK generates the release note line to be this:

Might be worth changing?
```
- Updated the Palo Alto Networks Security Advisories integration to %%UPDATE_CONTENT_ITEM_CHANGE_DESCRIPTION%%.
```